stackprinter/utils.py

```diff
@@ -37,8 +37,16 @@ def inspect_callable(f):
     except AttributeError:
         qname = None
 
-    filepath = code.co_filename
-    ln = code.co_firstlineno
+    try:
+        filepath = code.co_filename
+    except AttributeError:
+        filepath = None
+
+    try:
+        ln = code.co_firstlineno
+    except AttributeError:
+        ln = None
+        
```

Comment:
Hi, thanks for the contrib! Just two tweaks:
- could you shorten this to `getattr(code, 'co_filename', None)` and `getattr(code, 'co_firstlineno', None)`
- and add a comment that explains why this is done - i.e. in which situation these attributes can be None (under pypy, I understand?)

Reply:
![image](https://user-images.githubusercontent.com/2368879/193267536-7e2dcd4b-5e81-449b-b6ac-4c8e031ea678.png)